<html>
	<head>
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<link rel="stylesheet" type="text/css" href="css/reset.css">
		<link rel="stylesheet" type="text/css" href="css/professer_page.css">
		<link rel="stylesheet" type="text/css" href="css/csd.css">
		<link href="http://fonts.googleapis.com/css?family=Lato:100italic,100,300italic,300,400italic,400,700italic,700,900italic,900" rel="stylesheet" type="text/css">	
</head>
	
	<body>
		<div class="professor">
			<div class="container">
				<ul class="header">
					<li class="prof-img">
						<img class="clip-circle" src="artwork/filler_professor.jpg">
					</li>
					<li class="header-filler"></li>
					<li class="professor-info">
						<div class="prof-name">
							<span class="professor-title"></span> &nbsp;
							<span class="professor-first-name">Lila</span> &nbsp;
							<span class="professor-last-name">Kari</span>
						
							<div class="professor-position">Professor</div>
						</div>
					
						<table class="table-center">
							<tr>
						    	<td class="info-field">Email:</td>
								<td class="email">lila.kari@uwo.ca</td> 
							</tr>
							<tr>
						    	<td class="info-field">Tel: </td>
								<td class="tel">(519) 661-2111 <b> ext. </b> 86894</td> 
							</tr>
							<tr>
						    	<td class="info-field">Fax: </td>
								<td class="tel">(519) 661-3515 </td> 
							</tr>
							<tr>
						    	<td class="info-field">Office: </td>
								<td class="office">Middlesex College ###</td> 
							</tr>
						</table>
					</li>
					<li class="header-filler-2"></li>
				</ul>
			</div>
		</div>

		<div id="menu" class="menu-bar">
			<div class="container">
				<img class="arrow" src="artwork/arrow.png">
				<ul class="navigation">
				  	<li><a href="#">About</a></li>
				  	<li class="active-tab">Research</li>
				  	<li><a href="#">Teaching</a></li>
				</ul>
			</div>
		</div>

		<div class="container">
			<div class="box">
				<div class="box-header">
					Research Interests
				</div>
				<div class="box-content">
					<ul>
				    	<li>Natural Computation</li>
				    	<li>Bioinformation and biocomputation - theoretical aspects</li>
				    	<li>Nanocomputation by DNA self-assembly</li>
				    	<li>Models of cellular computation</li>
				    	<li>Watson-Crick complementarity in formal languages and automata theory</li>
					</ul>
				</div>
			</div>
		</div>

		<div class="container">
			<div class="box">
				<div class="box-header">
					Ph.D Students and Postdoctoral Fellows
				</div>
				<div class="box-content">
					<ul>
						<li>Bo Cui (Ph.D. 2011) - Microsoft Seattle</li>
						<li>Shinnosuke Seki (Ph.D. 2010) - Aalto University, Finland</li>
						<li>Elena Losseva (Ph.D. 2005) - Software Analyst Toronto</li>
						<li>Mark Daley (Ph.D. 2003) - Associate Professor and SHARCNET Chair at University of Western Ontario</li>
						<li>Zhi Xu (PDF) - Google</li>
						<li>David Doty (PDF) - Assistant Professor, University of California, Davis</li>
						<li>Ehsan Chinoforooshan (PDF) - Google</li>
						<li>Benoit Masson (PDF) - Directeur Pedagogique Regional Adjoint, Epitech, Rennes, France</li>
						<li>Elena Czeizler (PDF) - University of Helsinki, Finland</li>
						<li>Eugen Czeizler (PDF) - Aalto University, Finland</li>
						<li>Kalpana Mahalingam (PDF) - Assistant Professor at IIT Madras, India</li>
						<li>Fernando Sancho Caparrini (PDF) - Assistant Professor at University of Seville, Spain</li>
						<li>Yingwei Wang (PDF) - Assistant Professor at University of Prince Edward Island, Canada</li>
						<li>Petr Sosik (PDF) - Research Professor at Silesian University , Czech Republic, and Universidad Politecnica de Madrid, Spain. </li>
					</ul>
				</div>
			</div>
		</div>

		<div class="container">
			<div class="box">
				<div class="box-header">
					Selected Publications
				</div>
				<div class="box-content">
					<ul>
				        L.Kari, K.Hill, A.Sayem, R.Karamichalis, N.Bryans, K.Davis, N.Dattani. Mapping the space of genomic signatures. preprint pdf <br>
				        L.Kari, S.Kopecki. Deciding whether a regular language is generated by a splicing system. Proceedings of "DNA Computing and Molecular Programming", DNA18. In D.Stefanovic, A.Turberfield Eds., LNCS, vol.7433, 2012, 98-109. pdf <br>
				        El.Czeizler, Eu.Czeizler, L.Kari, S.Seki. An extension of the Lyndon-Schutzenberger result to pseudoperiodic words. Information and Computation, 209, 2011, 717-730. pdf <br>
				        L.Kari, S.Seki. An improved bound for an extension of Fine and Wilf's theorem, and its optimality. Fundamenta Informaticae, 101 (3), 2010, 215-236. (pdf) <br>
				        L.Adleman, J.Kari, L.Kari, D.Reishus, P.Sosik, The Undecidability of the Infinite Ribbon Problem: Implications for Computing by Self-Assembly. SIAM J. Comput. Volume 38, Issue 6, pp. 2356-2381 (2009) (pdf)<br>
						<br>
				        Self-assembly, the process by which simple objects autonomously come together to form complex structures, has recently become of great practical importance in the context of the developments in nanotechnolodgy and nanocomputation. A systematic study of self-assembly as a mathematical process has been initiated by Adleman. The individual components are modelled as square tiles on the infinite two-dimensional plane. Each side of a tile is covered by a specific ``glue'', and two adjacent tiles will stick iff they have matching glues on their abutting edges. Tiles that stick to each other may form various two-dimensional ``structures'' such as squares, rectangles, or may cover the entire plane. In this paper we focus on a special type of structure, called ribbon: A non-self-crossing sequence of tiles on the plane, in which successive tiles are adjacent along an edge, and abutting edges of consecutive tiles have matching glues. We namely prove that, given a finite set of tiles with glues (infinite supply of each tile type available) it is undecidable whether or not there exists an infinite ribbon that can be formed with these tiles. The result settles a decade-old open problem formerly known as the ``unlimited infinite snake problem''. An immediate consequence is the undecidability of existence of arbitrarily large structures self-assembled using tiles from a given tile set. <br>
				        L. Kari, S. Seki, On pseudoknot-bordered words and their properties, Journal of Computer and System Sciences, 75 (2009) 113-121. pdf <br>
				        L. Kari, K. Mahalingam, S. Seki, Twin-roots of words and their properties, Theoretical Computer Science, vol. 410, nr.24-25 (2009), 2393-2400. pdf <br>
				        L.Kari, G.Rozenberg. The many facets of natural computing. Communications of the ACM, Oct.2008, vol.51, no.10, 72-83 . Cover Story of Communications of the ACM. (pdf file) pdf file of the paper with no pictures but with extended reference list . <br>
				        E.Czeizler, L.Kari, S.Seki. On a Special Class of Primitive Words. MFCS 2008: 265-277 pdf <br>
				        L.Kari, K.Mahalingam, Involutively bordered words, International Journal of Foundations of Computer Science, 18(5) 2007, 1089-1106. pdf <br>
				        L.Kari, S.Konstantinidis, P.Sosik. On properties of bond-free DNA languages Theoretical Computer Science, vol. 334, no. 1-3 (2005), 131-159. pdf L. Kari, S. Konstantinidis, P. Sos√≠k: Bond-free Languages: Formalizations, Maximality and Construction Methods. International Journal of Foundations of Computer Science 16 (2005), 1039--1070. pdf<br>
				        <br>
				        The computation language of a DNA-based system consists of all the words (DNA strands) that can appear during any biocomputation of the system. In this paper we investigate properties of such languages which ensure that their words will not form undesirable bonds when used in DNA computations. The pioneering aspect and contribution of this research is to establish a link between classical coding theory and DNA computing. It turns out that the newly defined notions of DNA compliant and sticky-free languages are theoretically elegant generalizations of classical types of codes. Furthermore, the use of such languages as input data ensures that the DNA computations will proceed error-free, therefore their study has practical relevance for experimental DNA computing. <br>
				        L. Kari, S. Konstantinidis: Language Equations, Maximality and Error-detection. J. of Computer and System Sciences 70 (2005), 157-178. pdf <br>
				        Y.Wang, K.Hill, S.Singh, L.Kari. The spectrum of genomic signatures: From dinucleotides to Chaos Game Representation. GENE, 346 (2005), 173-185. pdf<br>
				        <br>
				        In the post genomic era, access to complete genome sequence data for numerous diverse species has triggered multiple avenues for examining and comparing primary DNA sequence organization of entire genomes. Previously, the concept of a genomic signature was introduced with the observation of species-type specific Dinucleotide Relative Abundance Profiles (DRAPs); dinucleotides were identified as the subsequence with the greatest bias in representation in a majority of genomes. Herein, we demonstrate that DRAP is one particular genomic signature contained within a broader spectrum of signatures. In this spectrum, an alternative genomic signature, Chaos Game Representation (CGR), provides a unique visualization of patterns in sequence organization. A genomic signature is associated with a particular integer order or subsequence length that represents a measure of the resolution or granularity in the analysis of primary DNA sequence organization. We quantitatively explore the organizational information provided by genomic signatures of different orders through different distance measures, including a novel Image Distance. The Image Distance and other existing distance measures are evaluated by comparing the phylogenetic trees they generate for 26 complete mitochondrial genomes from a diversity of species. The phylogenetic tree generated by the Image Distance is compatible with the known relatedness of species. Quantitative evaluation of the spectrum of genomic signatures may be used to ultimately gain insight into the determinants and biological relevance of the genome signatures. 
				        L.Kari, L.F.Landweber. Computational power of gene rearrangement. Proc. DNA 5, DIMACS Series, 54(2000), 207-216 pdf , ps<br>
				        <br>
				        Ciliates are a diverse group of a few thousand types of unicellular eukaryotes distinguished by the presence of two nuclei: an active macronucleus and a functionally inert micronucleus which contributes only to sexual reproduction. The macronuclear DNA forms after sexual reproduction from the micronuclear DNA by recombinations that accomplish a rearrangement of gene fragments in the desired order and the deletion of non-coding sequences. In this paper we develop a model for the guided homologous recombinations that take place during gene rearrangement and prove that it has the computational power of a Turing machine. This is a proof of concept indicating that unicellular organisms may have the capacity to perform any computation carried out by an electronic computer. <br>
				        L.Landweber, L.Kari. The evolution of cellular computing: nature's solution to a computational problem. Biosystems 52(1999) 3-13. pdf <br>
				        .Kari. DNA computing -- The arrival of biological mathematics. The Mathematical Intelligencer, 19(1997), 9--22. pdf <br>
				        L.Kari. On Insertion and Deletion in Formal Languages. Ph.D. Thesis, University of Turku, 1991. Cover page , Title page , ISBN , Acknowledgements , Part I , Part II . <br>
				        G.Paun, L.Santean. Parallel communicating grammar systems-- the regular case. Analele Universitatii din Bucuresti, Seria matematica-informatica,nr.2, 1989, 55-63. pdf <br>
					</ul>
				</div>
			</div>
		</div>

		<script src="http://code.jquery.com/jquery-1.11.2.min.js"></script>
		<script type="text/javascript">
		    var js = document.createElement("script"),
				link = document.createElement("link"),
				head = document.getElementsByTagName('head')[0];

			link.rel = "stylesheet";
	    	link.type = "text/css";
		    if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)) {
			    js.src = "js/mobile_prof.js";
		    	document.body.appendChild(js);

		    	link.href = "css/professer_page_mobile.css";
		    }
		    else
		    	link.href = "css/professer_page_desktop.css";

		    document.head.appendChild(link);
		</script>
	</body>
</html>